Source/NSImage+WebP.m

```diff
@@ -17,16 +17,7 @@ + (NSImage*)imageWithWebPData:(NSData*)data {
   int height = 0;
   UInt8 *webp = WebPDecodeRGBA( data.bytes, data.length, &width, &height );
   if (webp) {
-    UInt32 *fixup = (UInt32*)webp;
-    // If the alpha channel is 0, webP seems to take the view that it doesn't matter what's in it.
-    // Setting this to 0 for consistency with (e.g) loading from PNG.
-    for (int i = 0; i < width * height; i++ ) {
-      if (!(fixup[i] & 0xFF000000)) {
-        fixup[i] = 0x00000000;
-      }
-    }
-    
-    NSBitmapImageRep *rep = [[NSBitmapImageRep alloc] initWithBitmapDataPlanes:NULL pixelsWide:width pixelsHigh:height bitsPerSample:8 samplesPerPixel:4 hasAlpha:YES isPlanar:NO colorSpaceName:NSDeviceRGBColorSpace bitmapFormat:NS32BitLittleEndianBitmapFormat bytesPerRow:4 * width bitsPerPixel:32];
+    NSBitmapImageRep *rep = [[NSBitmapImageRep alloc] initWithBitmapDataPlanes:NULL pixelsWide:width pixelsHigh:height bitsPerSample:8 samplesPerPixel:4 hasAlpha:YES isPlanar:NO colorSpaceName:NSDeviceRGBColorSpace bitmapFormat:NS32BitLittleEndianBitmapFormat | NSAlphaNonpremultipliedBitmapFormat bytesPerRow:4 * width bitsPerPixel:32];
```

Comment:
The fix is to set the `NSAlphaNonpremultipliedBitmapFormat` option - which also means that we don't need the deleted code above.